src/NuGet.Clients/NuGet.PackageManagement.UI/PackageManagerToolWindowPane.cs

```diff
@@ -43,7 +43,7 @@ private void CleanUp()
         {
             if (_content != null)
             {
-                _content.CleanUp();
+                _content.Dispose();
```

Comment:
I can see you introduced `private bool _disposed = false;` in other other files. But not here, why?

Reply:
`PackageManagerToolWindowPane.cs` overrides `Dispose` method from the parent class and it doesn't implement `IDisposable` explicitly. The actual logic that checks whether the object has been disposed or not is addressed top in the inheritance hierarchy in [`WindowPane.cs`](http://index/?query=WindowPane&rightProject=Microsoft.VisualStudio.Help.Shell.14.0&file=WindowPane.cs&line=226).